### ISSUE TITLE
Network error: Unexpected token < in JSON at position 0

### ISSUE THREAD

**Issue by AlecMartin3** [Shopify/shopify-demo-app-node-react#30]
# Bug summary

Write a short description of the bug here ↓
I've followed the tutorial and got to "Build a resource list from a query" and I selected the three products as told by the tutorial. Unfortunately, it seems that ResourceListWithProducts fails to load and causes the error Network error: Unexpected token < in JSON at position 0. I then cloned this completed version and redid the setup steps and I got the same issue.

## Expected behavior

What do you think should happen?

ResourceListWithProducts loads without an error and the products are displayed correctly

## Actual behavior

What actually happens?

Tip: include an error message (in a `<details></details>` tag) if your issue is related to an error
an error at the bottom of the appeats that states:
<details> Network error: Unexpected token < in JSON at position 0 </details>
This has happened on "incomplete" and a fresh "complete" version of this tutorial

## Steps to reproduce the problem

For me, it was 
1. A fresh clone of this project
2. Setting up ngrok, creating a new app and getting the API keys.
3. Installing the new app 
4. npm install
5. npm run dev

## Specifications

- Browser: Chrome
- Device: Desktop
- Operating System: Windows

**Comment by keiraarts** [comment]
Hi - was this ever solved?

Running into a similar issue where my code works on ngrok, but not on a domain.

I believe it's because next.js or KOA can not read the route for /auth - but haven't a clue why.

**Comment by katiedavis** [comment]
@keiraarts issue was resolved but @AlecMartin3 are you still experiencing this? I've checkout all the branches today and updated master with a few unrelated changes but let me know if you're still seeing this. 

Do you have any console errors using the resource picker? Or is it just loading the list or products after selection?

**Comment by katiedavis** [comment]
I'm going to go ahead and close this issue as I haven't heard back in sometime.

**Comment by MladenJanjetovic** [comment]
I am getting the same error. It was working fine yesterday and today it says this on index page:

> Unexpected token < in JSON at position 0

Last thing I did was adding this
```<ResourceListWithProducts />```

And it worked. But now when server was reset I can't make it work again. 
I added new ngrok URLs like before. And the Annotated layout page is working fine, as well as Index page when nothing is selected via Resource Picker
